code/globals/overlays.dm

```diff
@@ -0,0 +1,6 @@
+/proc/create_overlay(from)
+	var/obj/overlay = new
+	overlay.icon = from?:worn_icon
+	overlay.icon_state = from?:worn_icon_state
+	overlay.layer = from?:layer
```

Comment:
don't use `:`, give `from` a type like `atom/from`.
also this proc should assume that `from` is never null, so don't use the `?`, if the proc is used incorrectly it should give an error.